hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneFileSystem.java

```diff
@@ -915,6 +915,16 @@ public Path createSnapshot(Path path, String snapshotName)
         OM_SNAPSHOT_INDICATOR + OZONE_URI_DELIMITER + snapshot);
   }
 
+  @Override
+  public void setTimes(Path f, long mtime, long atime) throws IOException {
+    incrementCounter(Statistic.INVOCATION_SET_TIMES, 1);
+    statistics.incrementWriteOps(1);
+    LOG.trace("setTimes() path:{}", f);
+    Path qualifiedPath = f.makeQualified(uri, workingDir);
+    String key = pathToKey(qualifiedPath);
```

Comment:
it doesn't call  ``checkPath(qualifiedPath);``, that Wrong FS thing must not be getting handled? Can you confirm if it is getting handled somewhere

Reply:
That's a good point. Looking at BasicOzoneFileSystem, most of the APIs do not call checkPath() or makeQualified(). Perhaps a separate jira to review the usage overall. What do you think?